
##  Conservation of regulatory elements with highly diverged sequences across large evolutionary distances

Identification of embryonic heart CREs in mouse and chicken

A synteny-based algorithm identifies ortholog genomic regions

IPP improves ortholog detection in distantly related species

Reanalysis of published CREs increases putative orthologs

Functional chromatin marks at IPP-projected CREs

A heart-specific support vector machine model validates IPP projections

TFBS conservation as an indicator of conserved CRE activity

IC enhancer pairs drive conserved expression patterns

IC CREs show higher TFBS shuffling

---

## Multiomic atlas with functional stratification and developmental dynamics of zebrafish cis-regulatory elements

The DANIO-CODE DCC

Transcript annotation and core promoter characterization

Developmental specialization of polymerase II gene promoters

Developmental dynamics and locus organization of enhancers

Functional conservation of epigenetic subdomains

---

## An AluYa5 Insertion in the 3′UTR of COL4A1 and Cerebral Small Vessel Disease

Baseline Characteristics of Study Participants

Clinical and Neuroimaging Features of Patients in Families F1 and F2

Linkage Analysis andWESData

WGS Data Analysis

AluYa5 Insertion in 3′UTR ofCOL4A1 and CSVD

Upregulation of COL4A1 mRNA and Protein in Carriers of AluYa5

AluYa5 Insertion and PAS Usage

Clinical and Neuroimaging Features of F3-F7 Patients

---


## Statistically and functionally fine-mapped blood eQTLs and pQTLs from 1,405 humans reveal distinct regulation patterns and disease relevance

An expanded catalog of fine-mapped whole-blood eQTLs

A fine-mapped pQTL resource from 1,384 EAS samples

Functional characterization of fine-mapped pQTLs

Contrasting eQTLs and pQTLs highlights distinct functional features

Buffering of blood mRNA expression variation in constrained genes

Limited colocalization between eQTLs and pQTLs

Examples of limited colocalization between eQTLs and pQTLs

pQTLs may be more relevant to complex traits than eQTLs

Characterizing the trans-pQTL effects

Buffering of context-specific (severe COVID-19) eQTL effects

Interpreting a negative correlation between mRNA and protein expression


Fine-mapping of eQTLs and other QTLs was performed based on our previous pipeline (that is, using FINEMAP v.1.3.1 and susieR v.0.11.43 with default parameters, where the inputs were the summary statistics and in-sample covariate-adjusted LD matrix), with two minor changes.

