
## Deep learning modeling of ribosome profiling reveals regulatory underpinnings of translatome and interprets disease variants

Overview of mRNABERT and benchmarks

Capturing multi-dimensional biological information of mRNA

Predicting ribosome load from 5′UTR sequences

Evaluating mRNABERT on CDS prediction tasks

Detecting RBP binding sites from 3′UTR sequences

Identifying m6A modification sites from 3′UTR sequences

Predicting splice sites and alternative polyadenylation

Applying mRNABERT to protein engineering tasks

Evaluating the applicability using full-length mRNA sequences


## Omnireg-gpt: a high-efficiency foundation model for comprehensive genomic sequence understanding

Overview of OmniReg-GPT: advancing long-sequence modeling through high-efficiency pretraining

Benchmark of genomics foundational models

OmniReg-GPT improves context-dependent gene expression prediction

OmniReg-GPT accurately predicts single-cell chromatin accessibility and transcription factor activity

OmniReg-GPT predicts 3D chromatin organization

OmniReg-GPT generates cell-type specific enhancers


## Riboformer: a deep learning framework for predicting context-dependent translation dynamics

Riboformer accurately clarifies ribosome density

Riboformer corrects experimental bias in unseen data

Riboformer identifies a limiting step in translation elongation

Riboformer identifies sequence determinants of ribosome collisions

Riboformer allows interpretation of exacerbated ribosome stalling in aging


## mRNABERT: advancing mRNA sequence design with a universal language model and comprehensive dataset

## A foundation language model to decipher diverse regulation of RNAs



