
## Deep learning modeling of ribosome profiling reveals regulatory underpinnings of translatome and interprets disease variants

Overview of mRNABERT and benchmarks

Capturing multi-dimensional biological information of mRNA

Predicting ribosome load from 5′UTR sequences

Evaluating mRNABERT on CDS prediction tasks

Detecting RBP binding sites from 3′UTR sequences

Identifying m6A modification sites from 3′UTR sequences

Predicting splice sites and alternative polyadenylation

Applying mRNABERT to protein engineering tasks

Evaluating the applicability using full-length mRNA sequences


## Omnireg-gpt: a high-efficiency foundation model for comprehensive genomic sequence understanding

Overview of OmniReg-GPT: advancing long-sequence modeling through high-efficiency pretraining

Benchmark of genomics foundational models

OmniReg-GPT improves context-dependent gene expression prediction

OmniReg-GPT accurately predicts single-cell chromatin accessibility and transcription factor activity

OmniReg-GPT predicts 3D chromatin organization

OmniReg-GPT generates cell-type specific enhancers


## Riboformer: a deep learning framework for predicting context-dependent translation dynamics

Riboformer accurately clarifies ribosome density

Riboformer corrects experimental bias in unseen data

Riboformer identifies a limiting step in translation elongation

Riboformer identifies sequence determinants of ribosome collisions

Riboformer allows interpretation of exacerbated ribosome stalling in aging


## Predicting functional constraints across evolutionary timescales with phylogeny-informed genomic language models
```
Alignment- and phylogeny-informed genomic language models
GPN-Star shows strong predictive capability for pathogenic coding variants
GPN-Star is the state-of-the-art predictor of pathogenic non-coding variants
GPN-Star is able to identify fine-mapped putatively causal variants in GWAS
Relevant evolutionary timescales for different variant effect prediction tasks
GPN-Star improves rare variant association testing
GPN-Star predictions are highly informative for complex trait heritability
Evolutionary timescales and the genetic architecture of complex traits
Tissue-specific predictions further improve heritability enrichment
GPN-Star learns functional elements in the genome and their dependencies
GPN-Star learns genome-wide evolutionary constraints
Controlling for mutation rate variation improves variant effect prediction
GPN-Star is a general and flexible framework for learning evolutionary constraints across diverse species and alignments
```

## mRNABERT: advancing mRNA sequence design with a universal language model and comprehensive dataset

## A foundation language model to decipher diverse regulation of RNAs



