
## Mutations in 329 probands with suspected renal electrolyte disorders

Mutations at non-GT/AG splice donor and acceptor sites in SLC12A3

Identification of small intragenic deletions of two or more exons in SLC12A3

Whole-genome analysis of 7 probands with monoallelic mutations IN SLC12A3

Deep intronic variants that disrupt splicing

Identification of a small intragenic deletion of two or more exons in SLC12A3

Rare intronic variant of unknown function

Maternally inherited hypomagnesemia caused by homoplasmic mitochondrial mutation

17q12 deletion in patients with hypomagnesemia

Biallelic mutations in other salt wasting genes

---

## Clinician-Driven Reanalysis of Exome Sequencing Data From Patients With Inherited Retinal Diseases

Patient Demographics

Diagnostic Performance of ES Reanalysis and Variation Spectrum

Factors Associated With New Molecular Diagnoses

Update of Clinical Diagnoses

Structural Variants

Mitochondrial Variants

Filtered or Not Captured Variants in ES

Noncanonical Splicing Variants in ES


---

## Rare disease gene association discovery in the 100,000 Genomes Project

Gene burden analytical framework

Application to 100KGP

Monogenic diabetes with UNC13A

Epilepsy with RBFOX3

Charcot–Marie–Tooth disease with ARPC3

Corneal abnormalities with POMK

Schizophrenia with GPR17



---

## Genetic association analysis of 77,539 genomes reveals rare disease etiologies

The Rareservoir

BeviMed infers 241 known and 19 unknown genetic associations

Variants in ERG are responsible for primary lymphoedema

Variants in PMEPA1 result in Loeys–Dietz syndrome

Variants in GPR156 lead to recessive congenital hearing loss

---

## Refinements and considerations for trio whole-genome sequence analysis when investigating Mendelian diseases presenting in early childhood

Comprehensive gene-agnostic trio WGS analysis pipeline for pediatric genetic diseases

Results from the application of the comprehensive analysis pipeline

Clinical impact and professional user experience

---


### The impact of inversions across 33,924 families with rare disease from a national genome sequencing project

Overall results summary

Different categories of SVs detected due to MantaINV call

Simple inversions

Inversions with loss (>10%) at one end

Deletions with small internal segment(s) retained

Duplication-triplications

Interlinked duplications

Other types of complex SV

Validation status

Retrospective analysis of exomes

ong-read genome sequencing

Support from RNA-seq studies

Final yield of likely diagnostic SVs

---

## Structural variant calling and clinical interpretation in 6224 unsolved rare disease exomes

## Clinical Variant Reclassification in Hereditary Disease Genetic Testing

Empirical Estimates of VC Accuracy

VUS Reclassifications and the Causes of Reclassification

Changes in VUS Reclassification Rates

Reversals in Classification and Reclassifications to VUS

Patterns of VUS Reclassifications in REA Groups

---


## Refinements and considerations for trio whole-genome sequence analysis when investigating Mendelian diseases presenting in early childhood

Comprehensive gene-agnostic trio WGS analysis pipeline for pediatric genetic diseases

Results from the application of the comprehensive analysis pipeline

Clinical impact and professional user experience


## Exome sequencing in undiagnosed congenital myopathy reveals new genes and refines genes–phenotypes correlations

Description of the cohort

Overall genetic characterization of the cohort

Main congenital myopathy genes and impact of exome sequencing

Known myopathy genes linked to classical phenotypes

Genes linked to atypical phenotypes

New genes implicated in myopathies

Widening the genetic and clinical heterogeneity




## Quantitative thresholds for variant enrichment in 13,845 cases: improving pathogenicity classification in genetic hearing loss

Samples and variants included in this study

Posterior probability and positive likelihood ratio values for varying strengths of evidence

PS4 assessment and refinement

Utility of the proposed PS4 thresholds in genetic testing



