## Use of whole genome sequencing to determine genetic basis of suspected mitochondrial disorders: cohort study

Demographics

Phenotype data

Diagnostic yield

Factors affecting diagnostic yield

Types of diagnosis

Potentially treatable disorders

Mitochondrial diagnoses compared with nonmitochondrial diagnoses

Muscle biopsy findings and mitochondrial complex activities

---

## Mutations in 329 probands with suspected renal electrolyte disorders

Mutations at non-GT/AG splice donor and acceptor sites in SLC12A3

Identification of small intragenic deletions of two or more exons in SLC12A3

Whole-genome analysis of 7 probands with monoallelic mutations IN SLC12A3

Deep intronic variants that disrupt splicing

Identification of a small intragenic deletion of two or more exons in SLC12A3

Rare intronic variant of unknown function

Maternally inherited hypomagnesemia caused by homoplasmic mitochondrial mutation

17q12 deletion in patients with hypomagnesemia

Biallelic mutations in other salt wasting genes

---

## Clinician-Driven Reanalysis of Exome Sequencing Data From Patients With Inherited Retinal Diseases

Patient Demographics

Diagnostic Performance of ES Reanalysis and Variation Spectrum

Factors Associated With New Molecular Diagnoses

Update of Clinical Diagnoses

Structural Variants

Mitochondrial Variants

Filtered or Not Captured Variants in ES

Noncanonical Splicing Variants in ES


---

## Rare disease gene association discovery in the 100,000 Genomes Project

Gene burden analytical framework

Application to 100KGP

Monogenic diabetes with UNC13A

Epilepsy with RBFOX3

Charcot–Marie–Tooth disease with ARPC3

Corneal abnormalities with POMK

Schizophrenia with GPR17



---

## Genetic association analysis of 77,539 genomes reveals rare disease etiologies

The Rareservoir

BeviMed infers 241 known and 19 unknown genetic associations

Variants in ERG are responsible for primary lymphoedema

Variants in PMEPA1 result in Loeys–Dietz syndrome

Variants in GPR156 lead to recessive congenital hearing loss

---

## Refinements and considerations for trio whole-genome sequence analysis when investigating Mendelian diseases presenting in early childhood

Comprehensive gene-agnostic trio WGS analysis pipeline for pediatric genetic diseases

Results from the application of the comprehensive analysis pipeline

Clinical impact and professional user experience

---


## The impact of inversions across 33,924 families with rare disease from a national genome sequencing project

Overall results summary

Different categories of SVs detected due to MantaINV call

Simple inversions

Inversions with loss (>10%) at one end

Deletions with small internal segment(s) retained

Duplication-triplications

Interlinked duplications

Other types of complex SV

Validation status

Retrospective analysis of exomes

ong-read genome sequencing

Support from RNA-seq studies

Final yield of likely diagnostic SVs

---

## Structural variant calling and clinical interpretation in 6224 unsolved rare disease exomes

## Clinical Variant Reclassification in Hereditary Disease Genetic Testing

Empirical Estimates of VC Accuracy

VUS Reclassifications and the Causes of Reclassification

Changes in VUS Reclassification Rates

Reversals in Classification and Reclassifications to VUS

Patterns of VUS Reclassifications in REA Groups

---


## Refinements and considerations for trio whole-genome sequence analysis when investigating Mendelian diseases presenting in early childhood

Comprehensive gene-agnostic trio WGS analysis pipeline for pediatric genetic diseases

Results from the application of the comprehensive analysis pipeline

Clinical impact and professional user experience

---

## Exome sequencing in undiagnosed congenital myopathy reveals new genes and refines genes–phenotypes correlations

Description of the cohort

Overall genetic characterization of the cohort

Main congenital myopathy genes and impact of exome sequencing

Known myopathy genes linked to classical phenotypes

Genes linked to atypical phenotypes

New genes implicated in myopathies

Widening the genetic and clinical heterogeneity


---

## Quantitative thresholds for variant enrichment in 13,845 cases: improving pathogenicity classification in genetic hearing loss

Samples and variants included in this study

Posterior probability and positive likelihood ratio values for varying strengths of evidence

PS4 assessment and refinement

Utility of the proposed PS4 thresholds in genetic testing


---

## SV catalogues of three major ancestry groups in Singapore

SG10K_Health SV Landscape

Impact of SVs on regulatory elements and gene bodies

SV patterns between international cohorts

SVs between Asian ancestry groups

SVs exhibit cis-linkage to disease GWAS loci

---



## Genomic Sequencing as a First-Tier Screening Test and Outcomes of Newborn Screening

Study Population

Overviewof the Genetic NBS and Follow-Ups

Patients Diagnosed With Disorders Screened by Both Genetic and Biochemical NBS

Newborns With Suspect Diseases

---

## Genome Sequencing as a Diagnostic Test in Children With Unexplained Medical Complexity

Phenotype and Family History Characteristics

Conventional Genetic Testing

Genome Sequencing Coverage and Analytical Validity

Primary Diagnostic Findings FromGenome Sequencing

Clinical Implications of Diagnostic Information

---

## Perspectives of Rare Disease Experts on Newborn Genome Sequencing

Respondent Characteristics

Perspectives on NBSeq

Free-Text Responses

Expert Concordance for Gene-Disease Associations

---


## Diagnostic Utility of Trio–Exome Sequencing for Children With Neurodevelopmental Disorders

Cohort Information

Diagnostic Yields in the Cohort

Spectrum of Candidate Variants

Spectrums of Diagnostic CNVs

Spectrums of Diagnostic SNVs


Data
Data available: No
Additional Information
Explanation for why data not available: Individual-level data are not publicly available due to
ethical and legal restrictions related to the Shanghai Children's Hospital.

---


## Exome copy number variant detection, analysis, and classification in a large cohort of families with undiagnosed rare genetic disease

CNV detection and analysis

Causal CNVs results

Evaluation of CMA coverage for each causal CNV

CNV confirmation

Special categories of CNVs

StrVCTVRE in silico score

CNV classification



A CNV is defined as high-confidence by GATK-gCNV (see Babadi et al.12 for details) if the following criteria are met:
(1)	The CNV is present in a high-quality sample (with ≤200 autosomal raw CNV calls of which at least 35 have QS >20).
(2)	The sample frequency of the call is ≤0.01 within the Broad callset.
(3)	The number of overlapped exons is ≥3.
(4)	The QS is equal or greater than the QS threshold as defined in Babadi et al.12 (QS >50 for duplications, >100 for deletions, and >400 for homozygous deletions).


## Genomic reanalysis of a pan-European rare-disease resource yields new diagnoses
Pan-European rare-disease data collection

New genetic diagnoses following systematic reanalysis

Cross-ERN analysis, recurrences and clinical actionability

---


## Whole-Exome Sequencing Among School-Aged Children With High Myopia

Cohort Information

Variational Spectrums Revealed for Known HM-Related Genes

Variant Yield by Severity of Myopia


Variant Interpretation

Variants were prefiltered so that only those passing the GATK VQSR (variant quality score recalibration) metric and those located outside of low-complexity regions remained. Low-certainty variant positions with a genotype depth (DP) less than 10 and genotype quality (GQ) less than 20 and heterozygous genotype calls with an allele balance of more than 0.8 or less than 0.2 were ignored. The annotation of variants was performed with Ensembl’s Variant Effect Predictor (VEP) version 99 for human genome assembly GRCh37.69We used the VEP70 CADD, LOFTEE,71 and SpliceAI plugins to generate additional bioinformatic estimations of variant deleteriousness. Potentially pathogenic protein-truncating variants (PTVs) were classified as frameshift variant, splice acceptor variant, splice donor variant, stop gained, or start lost variants. Variant filtering included the following steps: (1) the present analysis focuses on rare variants with presumed large effect sizes. Therefore, we excluded variants that had minor allele frequencies (MAF) higher than 0.005 from 3 external exome sequence databases (1000 Genomes; National Heart, Lung, and Blood Institute Exome Sequencing Project; and gnomAD) and our MAGIC cohort. (2) We excluded variants not consistent with hereditary patterns, which were only 1-hit heterozygous variants in autosomal recessive (AR) genes and homozygous variants in autosomal dominant (AD) genes. Annotation of pathogenic (P), likely pathogenic (LP), variants of uncertain significance (VUS), benign (B), and likely benign (LB) for potentially pathogenic PTVs were performed by using ANNOtate VARiation software version 2020June,72 and pathogenicity was assigned according to 2015 American College of Medical Genetics (ACMG) criteria using InterVar,73 which is a computational implementation of expert panel recommendations for clinical interpretation of genetic variants (ACMG 2015 criteria).74 A total of 75 genes were included in this study (eTable 2 in Supplement 2), including 16 nonsyndromic HM genes, 27 genes of eye syndromes associated with HM, and 32 genes of systemic syndromes associated withHM(hereafter, we refer to this gene list as HMgenes). All HMgenes were extracted from OMIM,75 IMI-Myopia Genetics Report,76 and PubMed for articles published up to December 2022 using genetic relevant keywords (for example, gene, genetic, mutation, or variant) and “high myopia.” Variants from 75 genes were selected from the WES data set of the school-aged population with HM.


