
## Plasma proteome variation and its genetic determinants in children and adolescents

Discovery and replication cohorts

Impact of demographic and health factors on plasma proteome

Effect of SNPs on the plasma proteome

Decomposition of variance in plasma protein levels

Characterization of pQTL effect sizes

Integrating pQTLs with variant–trait associations

Highly replicated pQTLs in children and adults



## Proteomic landscape of epithelial ovarian cancer

A proteomic landscape of Chinese EOCs

Malignancy-associated proteins in EOC tissue and plasma samples

Proteomic features among five histological subtypes of Chinese EOC

Proteins potentially correlated with prognosis of advanced HGSOC

Characterizing prognostic plasma proteins of primary HGSOC patients

Protein classifiers for prognosis prediction

Divergent DNA Damage Response in HGSOC: Primary vs. Relapsed


## Deep proteomic analysis of obstetric antiphospholipid syndrome by DIA-MS of extracellular vesicle enriched fractions

Deep coverage of serum proteomes through circulating EVs analysis

Establishing a workflow for proteomics of serum SEV and LEV-enriched fractions

Application of the workflow for proteomics of serum SEV and LEV enriched fraction to study OAPS

## Plasma proteome profiling reveals dynamic of cholesterol marker after dual blocker therapy

Cohort characteristics and research design

Plasma proteome revealed negative linkage between cholesterol metabolism and oncogenic signaling after DBT

Thyroid hormone related clinical indicators are significantly altered by DBT

Integrated analysis of proteome and radiology imaging reveals the potential of APOC3 as a biomarker in the DBT response

Integrated proteome-clinical features-based machine learning model provide an accurate measure for DBT



## Proteomic profiling of MIS-C patients indicates heterogeneity relating to interferon gamma dysregulation and vascular endothelial dysfunction

Patients included in the study

Overall architecture of pediatric SARS-CoV-2 and MIS-C plasma proteome

Characterizing pathways of activation in patients with MIS-C and SARS-CoV-2 infection

MIS-C is characterized by a disproportionate response to interferon gamma

MIS-C patients have an MAS-like phenotype characterized by IFNγ and CXCL9 signaling

PLA2G2A is a candidate biomarker for MIS-C and is associated with a thrombotic microangiopathy phenotype

Clinical heterogeneity among MIS-C patients


## Proteomic profiling reveals diagnostic signatures and pathogenic insights in multisystem inflammatory syndrome in children

Plasma proteins in children with MIS-C compared to controls

Diagnostic classifier for MIS-C using machine learning

Validation of the 4-protein diagnostic signature


## Longitudinal serum proteome mapping reveals biomarkers for healthy ageing and related cardiometabolic diseases

Serum proteome profiling of longitudinal cohorts

Longitudinal trajectories of serum proteome during follow-up

Identifying ageing-related proteins using longitudinal data

Biological implications of ageing-related proteins

Associations of ageing-related proteins with diseases

Ageing-related proteins as cardiometabolic health indicators

Determinants of PHAS and 22 ageing-related proteins




## Rare variant associations with plasma protein levels in the UK Biobank

UKB Pharma Plasma Proteome cohort characteristics

Variant-level associations

Assessing epitope effects

Gene-level protein abundance associations

Pan-ancestry collapsing analysis

Protein–protein interactions

Insights into allelic series

Biomarker discovery

Clonal haematopoiesis pQTLs

pQTL-augmented PheWAS


### Large-scale plasma proteomics comparisons through genetics and disease associations

Comparison of precision

Inter-platform correlation of levels

Dilution and subcellular location

Protein–phenotype associations

Detection of pQTLs

pQTL analysis by ancestry group

pQTL comparison between platforms

Relationship of pQTLs between platforms

Pleiotropic pQTLs

pQTLs and disease-associated variants




### Plasma proteomic associations with genetics and health in the UK Biobank
https://metabolomips.org/ukbbpgwas/


Data processing and quality control

Proteomic links with health and disease

Discovery of pQTLs

Replication of pQTLs

Non-European pQTL mapping

Comparisons with previous pQTL studies

Identification and fine-mapping of independent signals

SNP-based heritability of proteins

Protein interactions and pathways at trans loci

Insights into cytokine interactions

Complement cascade trans pQTL networks

Scaling of pQTL associations

Co-localization with expression QTLs

Drug targets and disease biology applications




### Blood protein assessment of leading incident diseases and mortality in the UK Biobank

Protein associations with incident outcomes

ProteinScore development

Exploration of the type 2 diabetes ProteinScore

Metabolomic score comparison




### Systematic discovery of gene-environment interactions underlying the human plasma proteome in UK Biobank

Discovery of variance QTLs underlying the plasma proteome

Variance QTLs largely overlap with main effect QTLs for the blood proteome

Variance QTLs unveil gene-environment interactions dispersed across the plasma proteome

GEIs implicate known biological pathways and are influenced by age and sex

Gene-environment interactions capture why some vQTLs lack genetic main effects on protein levels

Alternative explanations for variance QTLs underlying the plasma proteome




### Plasma proteomics for biomarker discovery in childhood tuberculosis

Clinical characteristics of the cohort

DIA-PASEF enabled high-throughput plasma proteomics

Identification of TB disease candidate biomarkers

Machine learning based identification of a TB biosignature

Detection of unconfirmed TB

### Associations of 2923 plasma proteins with incident inflammatory bowel disease in a prospective cohort study and genetic analysis

Participant characteristics

Observational associations of proteins with IBD

Mendelian randomization and colocalization analyses

Transcriptome analysis of protein-coding genes across different tissues

Single-cell type expression analysis of candidate protein-coding genes

PPI network, enrichment analysis, and druggability assessment



#### Statistical analysis
Baseline proteomics data were available for 53,022 participants, focusing on 49,443 participants of European ancestry. After excluding individuals with a history of IBD at baseline (N = 643), the final analysis comprised 48,800 individuals. We normalized individual protein levels for subsequent analyses. Baseline characteristics of the included participants were described using means (standard deviations) for continuous variables and counts (proportions) for categorical variables.

Cox proportional hazards regression models were used to estimate the associations between protein levels and incident IBD and its subtypes to calculate HR and 95% CI. Model 1 was unadjusted for covariates. Model 2 was initially adjusted for age, sex, assessment center, years of education, employment status, household income, and TDI. Model 3 was fully adjusted for smoking status, alcohol consumption, PA, healthy diet, sleep duration, and BMI. Furthermore, we performed sensitivity analyses. First, Cox analysis was performed after excluding samples with values below the limit of detection. Second, additional adjustments were made for the first 10 genetic principal components in the Cox analysis. Third, we excluded participants with a baseline history of other immune-mediated diseases (code details in Supplementary Table 7). Fourth, additional adjustments were made for baseline histories of major chronic conditions, including hypertension, diabetes, hyperlipidemia, cardiovascular diseases, cancer, and other immune-mediated diseases (code details in Supplementary Table 7), in the Cox analysis. Finally, the association between protein levels and the risk of developing IBD was analyzed based on the time of diagnosis (diagnosed within < 2, 2–6, and > 6 years of follow-up).

To develop protein-based prediction models for IBD and its subtypes, the dataset was first randomly split into an 80% training subset and a 20% testing subset. The significant proteins identified by Cox proportional hazards regression in 80% training subset were input into the training subset’s eXtreme Gradient Boosting (XGBoost) classifier69. The importance of these proteins was visualized using Shapley Additive Explanations (SHAP) values, and a simplified protein prediction model was constructed using the top 10 most important proteins to facilitate clinical application70. The testing dataset was then used to assess and compare the AUC values of the traditional risk factor model, simplified protein prediction model, and combined model.

Multiple testing corrections were applied using the FDR method with the “fdrtool” package, considering the number of proteins analyzed71. A threshold of FDR < 0.05 was used to determine statistical significance. This approach controls for the risk of false positives and ensures a more reliable, stringent interpretation of results.





## New biomarkers of Kawasaki disease identified by gingival crevicular fluid proteomics

Finally, MRM-MS confirmed the expression patterns of 12 DEPs (IFIT3, UB2L6, HP, A1AT, HSP90AA1, HNRPC, HSP90AB1, SAA1, MX1, B2M FKBP4, and TRAP1), thereby demonstrating high consistency with the DIA results and further validating the DEPs’ potential as biomarkers for KD.



